scipy/linalg/src/flinalg.pyf

```diff
@@ -0,0 +1,130 @@
+        end subroutine dlu_c
+        subroutine zlu_c(p,l,u,a,m,n,k,piv,info,permute_l,m1) ! in :_flinalg:lu.f
+            callprotoargument double*,complex_double*,complex_double*,complex_double*,int*,int*,int*,int*,int*,int*,int*
+            double precision dimension(m1,m1),intent(out),depend(m1) :: p
+            complex*16 dimension(m,k),intent(out),depend(m,k) :: l
+            complex*16 dimension(k,n),intent(out),depend(k,n) :: u
+            complex*16 dimension(m,n),intent(in,copy) :: a
+            integer, optional,intent(hide),depend(a) :: m=shape(a,0)
+            integer, optional,intent(hide),depend(a) :: n=shape(a,1)
+            integer, optional,intent(hide),depend(m,n) :: k=(m<n?m:n)
+            integer dimension(k),intent(hide),depend(k) :: piv
+            integer intent(out) :: info
+            integer, optional,intent(in) :: permute_l=0
+            integer, optional,intent(hide),depend(permute_l,m) :: m1=(permute_l?1:m)
+        end subroutine zlu_c
+        subroutine slu_c(p,l,u,a,m,n,k,piv,info,permute_l,m1) ! in :_flinalg:lu.f
+            callprotoargument float*,float*,float*,float*,int*,int*,int*,int*,int*,int*,int*
+            real dimension(m1,m1),intent(out),depend(m1) :: p
+            real dimension(m,k),intent(out),depend(m,k) :: l
+            real dimension(k,n),intent(out),depend(k,n) :: u
+            real dimension(m,n),intent(in,copy) :: a
+            integer, optional,intent(hide),depend(a) :: m=shape(a,0)
+            integer, optional,intent(hide),depend(a) :: n=shape(a,1)
+            integer, optional,intent(hide),depend(m,n) :: k=(m<n?m:n)
+            integer dimension(k),intent(hide),depend(k) :: piv
+            integer intent(out) :: info
+            integer, optional,intent(in) :: permute_l=0
+            integer, optional,intent(hide),depend(permute_l,m) :: m1=(permute_l?1:m)
+        end subroutine slu_c
+        subroutine clu_c(p,l,u,a,m,n,k,piv,info,permute_l,m1) ! in :_flinalg:lu.f
+            callprotoargument float*,complex_float*,complex_float*,complex_float*,int*,int*,int*,int*,int*,int*,int*
+            real dimension(m1,m1),intent(out),depend(m1) :: p
+            complex dimension(m,k),intent(out),depend(m,k) :: l
+            complex dimension(k,n),intent(out),depend(k,n) :: u
+            complex dimension(m,n),intent(in,copy) :: a
+            integer, optional,intent(hide),depend(a) :: m=shape(a,0)
+            integer, optional,intent(hide),depend(a) :: n=shape(a,1)
+            integer, optional,intent(hide),depend(m,n) :: k=(m<n?m:n)
+            integer dimension(k),intent(hide),depend(k) :: piv
+            integer intent(out) :: info
+            integer, optional,intent(in) :: permute_l=0
+            integer, optional,intent(hide),depend(permute_l,m) :: m1=(permute_l?1:m)
+        end subroutine clu_c
+    end interface 
+end python module _flinalg
+
+! This file was auto-generated with f2py (version:1.21.0).
```

Comment:
hmm, I don't think we want these `pyf` files?

Reply:
same answer, resolving.